main.py

```diff
@@ -1,12 +1,273 @@
+                video_id = int(video_id)
+                video_operations.selected_video = video_operations.get_one_video(video_id=video_id)
+                if video_operations.selected_video:
+                    print(f"Selected video: {video_operations.selected_video}")
+            else:
+                print("Id type is integer. Please enter valid id.")
+        
+        elif choice=='6':
+            print("Splendid! New customer more money! ")
+            name=input("What is the name of the customer ")
+            postal_code=input("What is the postal code? ")
+            phone=input("Please write down a phone number to contact ")
+            call_bar()
+            response = customer_operations.create_customer(name=name, postal_code=postal_code,phone=phone)
+            print_stars()
+            print(f"Here is the ID of new customer: {response['id']}")
+
+        elif choice=='7':
+            list_customer = customer_operations.get_all_customers()
+            for customer in list_customer:
+                print(f"Customer Id: {customer['id']}, name: {customer['name']}")
+            customer_id = input("Which customer would you like to update? Please enter ID: ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                print(f"Great! Let's update the customer with ID: {customer_id}")
+                name=input("What is the new name of your customer? ")
+                postal_code=input("What is the new postal code of your customer? ")
+                phone=input("Please write down a new phone number : ")
+                response = customer_operations.update_customer(customer_id, name=name, postal_code=postal_code,phone=phone)
+                print_stars()
+                print(f"Successfully updated the customer with ID: {response['id']} - name: {response['name']} - postal code: {response['postal_code']} - phone: {response['phone']}")
+            else:
+                print("Id type is integer. Please enter valid id.")
+
+        elif choice=='8':
+            list_customer = customer_operations.get_all_customers()
+            for customer in list_customer:
+                print(f"Customer Id: {customer['id']}, name: {customer['name']}")
+            customer_id = input("Which customer you would like to delete? Please enter ID: ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                customer_operations.delete_customer(customer_id)
+                print_stars()
+                print(f"Customer with ID {customer_id} has been deleted")
+            else:
+                print("Id type is integer. Please enter valid id.")
+
+        elif choice=='9':
+            print("Here are the customers:")
+            list_customers = customer_operations.get_all_customers()
+            for customer in list_customers:
+                print(f"Customer Id: {customer['id']}, name: {customer['name']}")
```

Comment:
This chunk of code is the same as lines 201-203 and is a great candidate for a helper function.